index.html

```diff
@@ -0,0 +1,182 @@
+              Addis Ababa on April 2023.
+            </p>
+          </div>
+
+          <div class="date-location">
+            <h2>2023.04.15(MON) ~ 20(FRI)</h2>
+            <h3>@ Millennium convention center, Addis Ababa</h3>
+          </div>
+        </div>
+        <img
+          class="friends"
+          src="assets/images/friend-no-bg.png"
+          alt="Friends cast Group picture"
+        />
+      </section>
+    </section>
+    <!-- Mid Section -->
+    <section class="main-program" id="program">
+      <h2>Main Events</h2>
+      <ul class="events">
+        <li class="event">
+          <img src="assets/images/keynote.svg" alt="keynote icon" />
+          <p class="event-title">Stories</p>
+          <p class="event-desc">
+            The producers will share a stories never told before on what goes on
+            behind the scene.
+          </p>
+        </li>
+        <li class="event">
+          <img src="assets/images/build.svg" alt="" />
+          <p class="event-title">Build</p>
+          <p class="event-desc">
+            Build competion of sets replica by top designers. Winners will get
+            big prizes.
+          </p>
+        </li>
+        <li class="event">
+          <img src="assets/images/expo.svg" alt="Expo Icon" />
+          <p class="event-title">Expo</p>
+          <p class="event-desc">
+            A tour of the studio and showcase of unique items that make the show
+            unforgetable.
+          </p>
+        </li>
+        <li class="event">
+          <img src="assets/images/forum.svg" alt="" />
+          <p class="event-title">Q&A</p>
+          <p class="event-desc">
+            You will get the chance to ask questions that were bugging you, all
+            cast on crew will be on stage to take your questions.
+          </p>
+        </li>
+        <li class="event">
+          <img src="assets/images/party.svg" alt="Party Icon" />
+          <p class="event-title">Party</p>
+          <p class="event-desc">
+            After party with the entire crew and producers, its gonna be hot and
+            you dont wann miss it.
+          </p>
+        </li>
+      </ul>
+      <button class="get-ticket">Get your ticket Here</button>
+    </section>
+    <!-- Guest section -->
+    <section class="guests" id="guests">
+      <h2>Featured Actors</h2>
+      <div class="guests-list"></div>
+    </section>
+
+    <section class="partner" id="partners">
+      <h2>Partner</h2>
+      <ul class="partners">
+        <li>
+          <img src="assets/images/warner-bros.jpeg" alt="Warner Brothers" />
+        </li>
+        <li><img src="assets//images/hbo.png" alt="Hbo max" /></li>
+        <li><img src="assets/images/netflix.png" alt="Netflix" /></li>
+        <li><img src="assets/images/apple-tv.png" alt="Apple" /></li>
+        <li><img src="assets/images/roku.png" alt="Roke" /></li>
+      </ul>
+    </section>
```

Comment:
I would recommend having the same background color in this section as in the Behance design. Kindly do so.

| Your design  | Behance design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/224688910-a7c7ed02-515f-45c4-8e59-58e122e2de48.png)| ![image](https://user-images.githubusercontent.com/33268927/224689028-5aed59e0-bfa7-4371-a480-97a408b90b28.png) |